### ISSUE TITLE
Updates made to template not reflecting on newly converted markdown files

### ISSUE THREAD

**Issue by JManyama** [Mermade/widdershins#415]
Hi, 

I believe I am missing a step possibly for when altering a template in Widdershins. The issue is that after removing the chevron symbol '»' from the schema table, newly converted files are still displaying the chevron.

I removed the chevron from these 3 locations to ensure it was not present anywhere in the template: 

widdershins/templates/openapi3/translations.dot/data.translations.indent = '»';
widdershins/templates/asyncapi1/translations.dot/data.translations.indent = '»';
widdershins/templates/semoasa/translations.dot/data.translations.indent = '»';

Then ran the following in the cli:  widdershins filename.Swagger.json -o newfile.md

The file converts, but with the chevron still showing.

I suspect I may be missing something in regard to the correct syntax to complete this type of step. I am new to Widdershins, could someone walk me through the correct steps to make this type of update?

Thanks.

**Comment by MikeRalphson** [comment]
The correct steps would be to copy the templates from `widdershins/templates/openapi3` (or which input file version you are using) to a new directory and edit the chevron out of `translations.dot` there. Then use the `--user-_emplates` option to use this directory when invoking widdershins.

Can you let me know if that works for you?

**Comment by JManyama** [comment]
I attached a video below of how I am running the command. I made a copy of the template, and also ensured that the copied 'templates' folder was in the directory for my application as well. When I run widdershins appending --user_templates to the end of the command, it still generates the markdown with the chevron. I'm probably still not entering this correctly. Thanks for the assistance here.

https://user-images.githubusercontent.com/74265784/104203204-ba66a180-53e0-11eb-8c06-e2d8d7a3fc84.mp4

](url)
